src/js/helpers/cleanData.js

```diff
@@ -0,0 +1,38 @@
+import getPathWithLangCode from 'js/helpers/language';
+
+export const cleanContacts = (contacts) => {
+
+  if(!contacts || !contacts.edges) return null;
+
+  return contacts.edges.map(({node: contact}) => {
+    let cleaned = Object.assign(contact.contact);
+    if(cleaned.hours && cleaned.hours.edges) {
+      cleaned.hours = cleaned.hours.edges.map(({ node: hours }) => hours);
+    }
+    return cleaned;
+  });
+};
+
+export const cleanRelatedServiceLinks = (links) => {
+
+  if (!links) return null;
+
+  return links.map((link) => {
+    return {
+      url: getPathWithLangCode(`/service/${link.slug}`),
```

Comment:
removed the getPathWithLangCode from some react modules since not all links are inbound links. It was goofy with 311 outbound links (see screenshot)
![screen shot 2018-01-12 at 4 46 06 pm](https://user-images.githubusercontent.com/3039125/34898419-a2f2cd80-f7b8-11e7-83cb-e763f4c20d28.png)

This will modify our url data during a cleanData step after data is fetched.